README.md

```diff
@@ -115,8 +115,10 @@ or by running the DPO triple generator on an existiting problem:
 python run_dpo_gen.jsonl
 ```
 
-Once you have the file `datasets/gen.jsonl`, you can run the DPO trainer with:
+Once you have the file `datasets/gen.jsonl`, you can run the DPO trainer with the following.
+You can set the CUDA\_VISIBLE\_DEVICES environment variable to whichever device number you want, informed by availability from nvidia-smi.
```

Comment:
can you put `nvidia-smi` in quotes?

Reply:
actually, you could also put `CUDA_VISIBLE_DEVICES` in quotes, then you wouldn't need to escape.